### PR TITLE
React graph work (don't review this one)

### PR DESCRIPTION
**Pull request by david-yz-liu**

### CODE REVIEW

**Comment by david-yz-liu** [comment]
@christinem @bell-kelly @R0ss @ryanfan @cchens @Ian-Stewart-Binks I'm looking to put the final touches on the react-graph branch before merging it in. We could use some testing here to find the last few bugs (feel free to submit pull requests to fix them as well).

Note: this branch is @ryanfan's work, I'm just putting the finishing touches on it.

File: hs/Response/Loading.hs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+largeLoadingIcon = H.div ! A.id "loading-icon" $ do
+              H.img ! A.id "c-logo" ! A.src "static/res/img/C-logo.png"
+              H.img ! A.id "compass" ! A.class_ "spinner" ! A.src "static/res/img/compass.png"
+
+{- Insert a small loading icon into the page -}
+smallLoadingIcon :: H.Html
+smallLoadingIcon = H.div ! A.id "loading-icon" $ do
+              H.img ! A.id "c-logo-small" ! A.src "static/res/img/C-logo-small.png"
+              H.img ! A.id "compass-small" ! A.class_ "spinner" ! A.src "static/res/img/compass-small.png"
+
```
Comment (Ian-Stewart-Binks) [inline hs/Response/Loading.hs]: Newline plus whitespace.

**Comment by Ian-Stewart-Binks** [comment]
Nice!

File: hs/Response/Loading.hs (hunk 1 above)
Comment (david-yz-liu) [inline hs/Response/Loading.hs]: @R0ss this one's on you - please fix. :)

**Pull request merged**